.travis.yml

```diff
@@ -1,11 +1,21 @@
 sudo: false
 services:
   - docker
+language: go
+go: 1.7
+env:
+  - CGO_ENABLED=0
+    GOOS=linux
 
+before_install:
+ - go get github.com/Masterminds/glide
+ - go install github.com/Masterminds/glide
+install:
+ - glide install
+ - go build -a -installsuffix cgo -v -ldflags='-s -w'
```

Comment:
@hinnerk because the "usual way" is made for self-contained binaries for `FROM scratch` images and we don't need that here